1-exercises/F-for-of-loop/exercise.js

```diff
@@ -10,7 +10,19 @@ let tubeStations = [
     "Oxford Street",
     "Tottenham Court Road"
 ];
+for(let element of tubeStations){
+    
+    console.log(element);
+
+}
+
 
 
 // TODO Use a for-of loop to capitalise and output each letter in the string seperately.
 let str = "codeyourfuture";
+for(let element of str){
+    const myArr=str.toUpperCase().split('');
```

Comment:
I think you should target 'element' here not 'str'. The aim of this exercise is to output each letter separately, not the whole string. Also, I don't think you need split method here because it unnecessarily converts you string back to an array.